libraries/rush-lib/src/cli/scriptActions/PhasedScriptAction.ts

```diff
@@ -379,6 +380,14 @@ export class PhasedScriptAction extends BaseScriptAction<IPhasedCommandConfig> {
         terminal.writeVerboseLine(`Incremental strategy: none (full rebuild)`);
       }
 
+      const { configuration: experiments } = this.rushConfiguration?.experimentsConfiguration;
+      if (
+        this.rushConfiguration?.packageManager === 'pnpm' &&
+        experiments?.usePnpmSyncForInjectedDependencies
+      ) {
+        new PnpmSyncCopyOperationPlugin().apply(this.hooks);
```

Comment:
```suggestion
        const { PnpmSyncCopyOperationPlugin } = await import('../../logic/operations/PnpmSyncCopyOperationPlugin');
        new PnpmSyncCopyOperationPlugin().apply(this.hooks);
```

Reply:
Are we really need to move the import statement to here? I saw other plugins, like `CacheableOperationPlugin` and `LegacySkipPlugin`, they are also called conditional, but the import statement is on top.

Reply:
This affects the Webpack bundling of Rush.  To understand the answer, you need to examine the effect on the bundle.